gcsfs/tests/test_core.py

```diff
@@ -1478,3 +1479,16 @@ def test_find_maxdepth(gcs):
 
     with pytest.raises(ValueError, match="maxdepth must be at least 1"):
         gcs.find(f"{TEST_BUCKET}/nested", maxdepth=0)
+
+
+def test_mkdir_options(gcs):
+    gcs = GCSFileSystem(endpoint_url=gcs._endpoint)
+    if not gcs.on_google:
+        pytest.skip("emulator doesn't support IAM policies.")
```

Comment:
Instead, we should mock to make sure that at least the right IAM config dict is being sent